### DISCUSSION TITLE
Overview session on did:iden3 / Polygon ID

### DISCUSSION THREAD

**Discussion by ottomorac** [veramolabs/did-eth#14] [Ideas]
**Hello All,**

As part of the study of current innovations out there in the identity space we would like to be able to provide an overview of the did method used by Polygon ID, called did:iden3. The idea would be to schedule a session for the end of January to provide an overview of our approach. 

Here is a public presentation on Polygon ID:   https://pitch.com/public/1b7b59e6-4c5e-4504-b58d-8bb7ffbd27e1

How about we schedule this for the session on Thursday 26-January?

Thoughts?

Otto

**Comment by strumswell** [comment]
Would love to hear more about `did:iden3` as I have not heard a lot about it yet. I am sure there are a few learnings we can leverage!

**Comment by ottomorac** [comment]
Link to presentation on Polygon ID that took place last week: https://www.youtube.com/watch?v=WNJbTfD8ong
